app/src/main/java/net/squanchy/service/firestore/model/schedule/FirestoreSchedule.kt

```diff
@@ -34,8 +35,14 @@ class FirestorePlace {
 class FirestoreTrack {
     lateinit var id: String
     lateinit var name: String
+    @JvmField
+    @field:PropertyName("accent_color")
```

Comment:
These are encoded in snake_case on the DB.

Reply:
Plus in general proguard is happier with these